.circleci/config.yml

```diff
@@ -1,14 +1,76 @@
-# Golang CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  build:
-    docker: # use the docker executor type; machine and macos executors are also supported
-      - image: circleci/golang:1.9
+  sanity-check:
+    working_directory: /go/src/github.com/ligato/networkservicemesh
+    docker:
+      - image: circleci/golang:1.10
+    steps:
+      - checkout
+      - run: sudo apt-get install yamllint shellcheck python3-pkg-resources
+      - run: yamllint -c .yamllint.yml $(git ls-files '*.yaml' '*.yml' | grep -v 'vendor/')
```

Comment:
If these sanity-checks fail, does the build itself fail as well and stop here? I assume it does, just making sure, because this is the behavior we will want.

Reply:
Yes, if sanity-check fail the build itself will fail and stop here.